.github/workflows/docs-checks.yml

```diff
@@ -1,109 +0,0 @@
-name: Check Docs
-# https://github.com/marketplace/actions/sphinx-build
-
-on:  # Trigger the workflow on push or pull request, but only for the master branch
-  push:
-    branches: [master, "release/*"]
-  pull_request:
-    branches: [master, "release/*"]
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
-  cancel-in-progress: ${{ ! (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release/')) }}
-
-jobs:
-  doctest:
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          submodules: true
-      - uses: actions/setup-python@v2
-        with:
-          python-version: 3.9
-
-      - name: Reset caching
-        run: python -c "import time; days = time.time() / 60 / 60 / 24; print(f'TIME_PERIOD=d{int(days / 2) * 2}')" >> $GITHUB_ENV
-
-      # Note: This uses an internal pip API and may not always work
-      # https://github.com/actions/cache/blob/master/examples.md#multiple-oss-in-a-workflow
-      - name: Cache pip
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-docs-test-pip-td${{ env.TIME_PERIOD }}-${{ hashFiles('requirements/pytorch/*.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-docs-test-pip-td${{ env.TIME_PERIOD }}-
-
-      - name: Install dependencies
-        env:
-          PACKAGE_NAME: pytorch
-          FREEZE_REQUIREMENTS: 1
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y cmake pandoc
-          pip --version
-          pip install -q fire
-          # python -m pip install --upgrade --user pip
-          pip install -e . --quiet -r requirements/pytorch/docs.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
-          pip install -r requirements/pytorch/devel.txt
-          pip list
-        shell: bash
-
-      - name: Test Documentation
-        env:
-          SPHINX_MOCK_REQUIREMENTS: 0
-        working-directory: ./docs
-        run: |
-          # First run the same pipeline as Read-The-Docs
-          make doctest
-          make coverage
-
-  make-docs:
```

Comment:
Removing `make-docs` since this is a complete duplicate of the CircleCI job building docs.

Reply:
tbh, if we find a way how to share/browse artefact from GHA, I would rather drop CircleCI and keep in-house

Reply:
> I would rather drop CircleCI 

@Borda Found a possible alternative #13572 to CircleCI :]

Reply:
Resolving this comment as the removal of doctest has been reverted.